src/commands/client-kill.json

```diff
@@ -41,77 +41,90 @@
         ],
         "arguments": [
             {
-                "name": "ip:port",
-                "type": "string",
-                "optional": true
-            },
-            {
-                "token": "ID",
-                "name": "client-id",
-                "type": "integer",
-                "optional": true,
-                "since": "2.8.12"
-            },
-            {
-                "token": "TYPE",
-                "name": "normal_master_slave_pubsub",
+                "name": "filter",
                 "type": "oneof",
-                "optional": true,
-                "since": "2.8.12",
                 "arguments": [
                     {
-                        "name": "normal",
-                        "type": "pure-token",
-                        "token": "normal"
-                    },
-                    {
-                        "name": "master",
-                        "type": "pure-token",
-                        "token": "master",
-                        "since": "3.2.0"
-                    },
-                    {
-                        "name": "slave",
-                        "type": "pure-token",
-                        "token": "slave"
+                        "name": "ip:port",
+                        "type": "string",
+                        "deprecated_since": "2.8.12"
                     },
                     {
-                        "name": "replica",
-                        "type": "pure-token",
-                        "token": "replica",
-                        "since": "5.0.0"
-                    },
-                    {
-                        "name": "pubsub",
-                        "type": "pure-token",
-                        "token": "pubsub"
+                        "name": "new-format",
+                        "type": "oneof",
+                        "multiple": true,
```

Comment:
"oneof" and "multiple" cannot go together (maybe add a verification for that in the python script?)

Reply:
i think this should be of type "block" (it's not "oneof" because i can use ADD and LADDR together for example)

Reply:
the "block" is a part of the bigger "oneof" (which is somewhat similar to optional)

i think each arg inside the block should be "multiple"

Reply:
It's unclear (based on the available docs) if `multiple: true` is can be zero occurrences. If that is the case using multiple could work except for `normal_master_slave_pubsub`. But I guess it would make sense to just make that `optional: true`.

If that is not the case, can we make them `multiple: true` and `optional: true` at the same time? I am not sure where to look up those restrictions.


```
{
    "name": "new-format",
    "type": "block",
    "arguments": [
        {
            "token": "ID",
            "name": "client-id",
            "type": "integer",
            "multiple": true,
            "since": "2.8.12"
        },
        {
            "token": "TYPE",
            "name": "normal_master_slave_pubsub",
            "optional": true,
            "type": "oneof",
            "since": "2.8.12",
            "arguments": [
                {
                    "name": "normal",
                    "type": "pure-token",
                    "token": "normal"
                },
                ...
            ]
        },
        ...
    ]
}
                    ```

Reply:
@valkum sorry i was mistaken: it is possible to combine "oneof" with "multiple" - it's just hard to render the command syntax from it :)

anyway, LGTM, please `make` so that `commands.c` will be updated. push and we'll merge.